frontend/src/metabase/routes-public.jsx

```diff
@@ -6,12 +6,14 @@ import { Route } from "metabase/hoc/Title";
 import PublicNotFound from "metabase/public/components/PublicNotFound";
 
 import PublicApp from "metabase/public/containers/PublicApp";
+import PublicAction from "metabase/public/containers/PublicAction";
 import PublicQuestion from "metabase/public/containers/PublicQuestion";
 import PublicDashboard from "metabase/public/containers/PublicDashboard";
 
 export const getRoutes = store => (
   <Route title={t`Metabase`}>
     <Route path="public" component={PublicApp}>
+      <Route path="action/:actionId" component={PublicAction} />
```

Comment:
This should be a :uuid, like the others
```suggestion
      <Route path="action/:uuid" component={PublicAction} />
```

Reply:
Yup, I'm planning to replace these once the BE is ready. It's probably to early to review this